setup.py

```diff
@@ -78,6 +78,8 @@ def get_cryptography_version():
         "kubernetes_asyncio",
         "pydantic",
         "asyncstdlib",
+        "typing_extensions",
+        "backports.cached_property",
```

Comment:
This won't do the same thing as ``cached_property``, I'm afraid. The ``lru_cache`` will keep the instance alive once its property is cached; if we go for a limited cache size, the property will get recomputed every now and then – that could e.g. lead to dropping an ``asyncio.Queue`` and all its items.  
Practically, I don't think ``property``+``lru_cache`` will be a problem for how HTCondor now uses the cache. Since it is a more general helper, I would like to avoid adding technical debt here, though.

A possible alternative is to roll our own, it's pretty simple and will be gone once we hit Python 3.8.

Reply:
Okay, I see. We can continue to use `backports.cached_property` then.